### DISCUSSION TITLE
Low Total Reading and Low Phase R

### DISCUSSION THREAD

**Discussion by BlipRanger** [emporia-vue-local/esphome#289] [Q&A] [answered]
This may be more of an installation issue, but my phase R values are always between -50 and 50 basically and my overall (when compared to the meter data) is up to 3x lower than it should be. Is this software/config issue on my part or have I installed something wrong?

**Comment by gekkehenkie11** [comment]
Not sure what you're exactly saying, what is "phase R"? You mean "Red"? So you're talking about 1 of the main phases?  You're saying that your power values for your main clamps are off? This happens with stock firmware too then, I assume? You did install in them in the right location, where utility enters your building? Maybe a photo of your setup? And your YAML file?

**Comment by gekkehenkie11** [comment]
Anyway I understand, from reading the VUE3 thread, that you're using a VUE3. Just like that last guy there noted, I read there that you might need to change the CT port order, try that, so swapping the main CT's to a different port.

**Reply by BlipRanger** [reply, marked as answer]
> Anyway I understand, from reading the VUE3 thread, that you're using a VUE3. Just like that last guy there noted, I read there that you might need to change the CT port order, try that, so swapping the main CT's to a different port.

Yep, that's what it was: https://github.com/emporia-vue-local/esphome/discussions/264#discussioncomment-9713465
Thank you.
